### PR TITLE
Adding processing the empty non system ks.cf list

### PR DESCRIPTION
**Pull request by aleksbykov**
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I gave variables/functions meaningful self-explanatory names
- [ ] I didn't leave commented-out/debugging code
- [ ] I didn't copy-paste code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -1005,9 +1005,11 @@ def _parse_toppartitions_output(output):
 
         def generate_random_parameters_values():  # pylint: disable=invalid-name
             ks_cf_list = get_non_system_ks_cf_list(self.loaders.nodes[0], self.cluster.nodes[0])
-            if not ks_cf_list:
-                raise Exception("No user defined keyspace.columnfamily")
-            ks, cf = random.choice(ks_cf_list).split('.')
+            try:
+                ks, cf = random.choice(ks_cf_list).split('.')
+            except IndexError:
+                self.log.error('User-defnied Keyspace and ColumnFamily are not found %s.', ks_cf_list)
```
Comment (bentsi) [inline sdcm/nemesis.py]: `defnied` -> `defined`

File: sdcm/nemesis.py [hunk 2]
```diff
@@ -1005,9 +1005,11 @@ def _parse_toppartitions_output(output):
...
+                raise Exception('User-defnied Keyspace and ColumnFamily are not found')
```
Comment (bentsi) [inline sdcm/nemesis.py]: need exception details and more debugging to log in order to understand why we fail

**Comment by bentsi** [comment]
@aleksbykov please address comments!

File: sdcm/nemesis.py (hunk 1 above)
Comment (aleksbykov) [reply sdcm/nemesis.py]: fixed

File: sdcm/nemesis.py (hunk 2 above)
Comment (aleksbykov) [reply sdcm/nemesis.py]: Adding details if error happened to log and to raising Exception

**Pull request merged**